### PR TITLE
Add lesson 19

### PR DESCRIPTION
**Pull request by Crushford**

### CODE REVIEW

File: lesson19.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+---
+
+### Solutions for previous lesson
+
+---
+
+### The Kitchen Storage Tracker
+
+Imagine you are designing an app to track the storage in your Pizza and Pasta Restaurant.
+The function you must write will accept 2 parameters;
```
Comment (Crushford) [inline lesson19.md]:
```suggestion
The function you must write will accept 2 inputs;
```

**Review by carlotrimarchi — APPROVED** [review]

**Comment by Crushford** [comment]
great, thanks for fixing those mistakes @carlotrimarchi

File: lesson19.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
+<!-- .slide: id="lesson19" -->
+
+# Basic Frontend - Spring 2020
+
+Lesson 19, Thursday, 2020-5-14
+
+---
+
+#### A word from Sevtap
+
```
Comment (haraldF) [inline lesson19.md]:
```suggestion
```

File: lesson19.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
+<!-- .slide: id="lesson19" -->
+
+# Basic Frontend - Spring 2020
+
+Lesson 19, Thursday, 2020-5-14
```
Comment (haraldF) [inline lesson19.md]:
```suggestion
Lesson 19, Thursday, 2020-05-14
```

**Review by haraldF — APPROVED** [review]

**Pull request merged**